src/include/daos_srv/rsvc.h

```diff
@@ -46,12 +47,24 @@ struct ds_rsvc;
 
 /** Replicated service class */
 struct ds_rsvc_class {
+	/** Name of the service class. */
+	const char *const sc_classname;
+
```

Comment:
It seems sc_name is sufficient, no? Also, we're interested in which particular instance of pool service class, for instance, fails to start or stop, I think.